lib/rubocop/cop/rspec/sort_tags.rb

```diff
@@ -0,0 +1,333 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Sort RSpec tags alphabetically.
+      #
+      # @example EnforcedStyle: sort_only (default)
+      #   # bad
+      #   describe 'Something', :b, :a
+      #   context 'Something', foo: 'bar', baz: true
+      #   it 'works', :b, :a, foo: 'bar', baz: true
+      #
+      #   # good
+      #   describe 'Something', :a, :b
+      #   context 'Something', baz: true, foo: 'bar'
+      #   it 'works', :a, :b, baz: true, foo: 'bar'
+      #
+      # @example EnforcedStyle: hash
+      #   # bad
+      #   describe 'Something', :b, :a
+      #   context 'Something', foo: 'bar', baz: true
+      #   it 'works', :b, :a, foo: 'bar', baz: true
+      #
+      #   # good
+      #   describe 'Something', a: true, b: true
+      #   context 'Something', baz: true, foo: 'bar'
+      #   it 'works', a: true, b: true, baz: true, foo: 'bar'
+      #
+      # @example EnforcedStyle: prefer_symbols
+      #   # bad
+      #   describe 'Something', :b, :a
+      #   context 'Something', foo: 'bar', baz: true
+      #   it 'works', :b, :a, foo: 'bar', baz: true
+      #
+      #   # good
+      #   describe 'Something', :a, :b
+      #   context 'Something', :baz, foo: 'bar'
+      #   it 'works', :a, :b, :baz, foo: 'bar'
+      #
+      class SortTags < Base
+        extend AutoCorrector
+        include ConfigurableEnforcedStyle
+        include RangeHelp
+
+        HOOKS = %i[
+          prepend_before
+          before
+          append_before
+          around
+          prepend_after
+          after
+          append_after
+        ].to_set.freeze
+
+        # Examples, ExampleGroups, SharedGroups, and Hooks
+        RESTRICT_ON_SEND = %i[
+          after
+          append_after
+          append_before
+          around
+          before
+          context
+          describe
+          example
+          example_group
+          fcontext
+          fdescribe
+          feature
+          fexample
+          ffeature
+          fit
+          focus
+          fscenario
+          fspecify
+          it
+          its
+          pending
+          prepend_after
+          prepend_before
+          scenario
+          shared_context
+          shared_examples
+          shared_examples_for
+          skip
+          specify
+          xcontext
+          xdescribe
+          xexample
+          xfeature
+          xit
+          xscenario
+          xspecify
+        ].to_set.freeze
```

Comment:
Wondering if this would work if someone decides to configure `RSpec/Language` and add e.g. an alias to an example,  e.g. `this { fails }`.

Reply:
I added a spec for configured language. Does that meet your requirements?

Reply:
Muy bien!